app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="title">Welcome to Adopt Don't Shop</h1>
+
```

Comment:
welcome page will need extra styling, hope to add background image to welcome page